.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Join the Discord for support.
+    url: https://discord.quiltmc.org
+    about: Discuss Quilt Standard Libraries on our Discord server.
```

Comment:
Why does this template exist?

Reply:
this isn't a "template" per se, it just makes it so issues have to be made using a template

Reply:
here, you don't see "Don’t see your issue here? Open a blank issue."
![image](https://github.com/user-attachments/assets/364e0aa9-3cd1-4b51-9873-a6d77c80c493)

whereas here, you do see "Don’t see your issue here? Open a blank issue."
![image](https://github.com/user-attachments/assets/ff06f658-e8d6-4115-88c2-8adb1cac3ce8)

Reply:
Ah. I still opening a blank issue should be allowed though

Reply:
personally, i think its better for organization to make people use the issue templates and issue tags can be automatically applied on creation

Reply:
Yes but there are always extra cases. such as documentation or toolchain.

Reply:
well, if we do allow people to make issues without using a template, it should be specified to not use it for things the issue templates cover

Reply:
Well yes, thats why the templates need to be clear so that people default to those instead of a blank issue